app/routes/content.js

```diff
@@ -6,6 +6,15 @@ import config from 'ember-get-config';
 import loadAll from 'ember-osf/utils/load-relationship';
 import permissions from 'ember-osf/const/permissions';
 
+// Error handling for API
+const handlers = new Map([
+    // format: ['Message detail', 'page']
+    ['Authentication credentials were not provided.', 'forbidden'], // 401
+    ['You do not have permission to perform this action.', 'forbidden'], // 403
```

Comment:
Definitely long-term 401 should bounce to CAS login, which has some implied functionality for this helper. But for now, the above is okay. For the 403, since the original ticket did have Not Found and it was approved in a Product Demo, either get Product to approve this as is (pointing out the original ticket) or go with the original version. I'm good either way. It should probably be demoed to Product regardless.